chain/client/src/client.rs

```diff
@@ -65,6 +68,16 @@ pub struct ClientActor {
     num_blocks_processed: u64,
     /// Total number of transactions processed.
     num_tx_processed: u64,
+
+
+}
+
+fn wait_until_genesis(genesis_time: &DateTime<utc>) -> Result<(), OutOfRangeError> {
+        let now : DateTime<Utc> = Utc::now();
+        let duration = now.signed_duration_since(*genesis_time);
+        let std_duration = duration.to_std()?;
```

Comment:
hm... not actually sure what's the best way here, may be `Duration::from_seconds(duration.seconds())` ?

Reply:
I think,  chrono::Duration has to be converted to std::time::Duration in order to be passed to thread::sleep as a parameter.
to_std() method actually resolves this issue.

Reply:
From my understanding `Duration::from_seconds(duration.seconds())` does the same and doesn't raise error. Am I wrong?